coreTable/OwidTable.ts

```diff
@@ -266,24 +272,29 @@ export class OwidTable extends CoreTable<OwidRow, OwidColumnDef> {
         }, `Filter out countries with population less than ${minPop}`)
     }
 
-    filterByTargetTime(targetTime: Time, tolerance: Integer) {
+    filterByTargetTimes(targetTimes: Time[], tolerance: Integer = 0) {
```

Comment:
I'm not seeing this used yet. Is this in anticipation of scatters?

Reply:
Yeah, tables & slope charts, where you have two points – start & end.

Reply:
Coming in `tolerance-charts` but left the utils in this one